pyproject.toml

```diff
@@ -58,7 +58,8 @@ test = [
     "pytest-timeout",
     "pytest-console-scripts",
     "ipykernel",
-    "jupyterlab_server[test]>=2.13,<3",
+    "jupyter_server[test]>=2.0.0rc3,<3",
```

Comment:
This addition seems to be necessary to fix https://github.com/jupyterlab/jupyterlab/issues/13438 (probably related to the pre-release).